speechpy/processing.py

```diff
@@ -30,8 +30,11 @@ def stack_frames(sig, sampling_frequency, frame_length=0.020, frame_stride=0.020
     frame_stride = float(np.round(sampling_frequency * frame_stride))
 
     # Check the feasibility of stacking
-    if length_signal <= frame_sample_length:
+    if length_signal < frame_sample_length:
+        return np.empty((0, frame_sample_length))
```

Comment:
Thank you so much for your pull request. I investigated this case. Don't you think the implementation of the following:

`    
  if length_signal <= frame_sample_length:
         signal = np.lib.pad(sig, (0, frame_sample_length-length_signal),'constant',constant_values=(0, 0))
         numframes = 1
`

is better?

Reply:
That adds zero padding rather than returning a whole empty vector.

Reply:
I didn't realize that there was alternate padding behavior depending on `zero_padding`. Because of this, I think it makes more sense to just treat a small signal like a normal one, which means if you pass `zero_padding=True`, it will behave just written above. I've amended the commit so that all it does is remove the special logic for signals smaller than `frame_sample_length`.